ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -38,20 +32,17 @@
 
   private final Config config;
   private final RunnerBuilder runnerBuilder;
+  private final TransactionSigner signer;
 
-  public EthSigner(final Config config, final RunnerBuilder runnerBuilder) {
+  public EthSigner(
```

Comment:
nit; `TransactionSigner` can have the `final` modifier

Reply:
done